app/models/pageflow/published_entry.rb

```diff
@@ -25,6 +25,7 @@ class PublishedEntry
              :author, :publisher, :keywords,
              :theme,
              :password_protected?,
+             :created_at, :updated_at,
```

Comment:
Since `Revision` is `belongs_to :entry, touch: true`, both are probably safer to delegate to `Entry`, right?